README.md

```diff
@@ -1 +1,193 @@
-# Ruby-Capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+ <!-- LOGO -->
+
+  <img src="./assets/catalogs.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+<!-- MAIN HEADING -->
+
+  <h3><b>Catalog App</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- INTRO -->
+# 📖 Catalog App <a name="about-project"></a>
+
+> This is a web application that helps to keep a record of different types of things you own: books, music albums, movies, and games.
+
+## 🛠 Built With <a name="built-with"></a>
+1. Ruby
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> - Keep catalog of books
+> - Keep catalog of music albums
+> - Keep catalog of games
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+LIVE DEMO
+
+> Watch a [live](https://drive.google.com/file/d/1itp4gndEFGNHYehVzEFODrozzD4FntTY/view?usp=sharing) demo of the project.
```

Comment:
- [ ] [REQUIRED] Good job so far! I appreciate what you have done but your `Presentation` video's  maximum length of the video should be `5 minutes`✔️  but your's contains `8:15 mins`❌. Please implement my suggestion 🙏🏼

![image](https://github.com/anyars-encarta/Ruby-Capstone/assets/63915024/1b62eda7-d542-483b-bb5b-c4fef9a83eba)